### ISSUE TITLE
Exception when code is minified

### ISSUE THREAD

**Issue by nevf** [schrotie/shadow-query#2]
Line 461 ```if(nodes.name == 'processDynNodes')``` does not work when the module is minified because ```name``` is mangled by the minifier.  

This is explained in: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name#JavaScript_compressors_and_minifiers

In essence you can't reliably use ```Function.name``` in production code.

My hacky fix is:
````
function dynTemplate(template) { 
/*return */ function processDynNodes(parent, callback) { ... }
    processDynNodes._name = 'processDynNodes';
    return processDynNodes;
}
````

and in:
````
function toNodes(parent, nodes, callback) {
...
	if(nodes._name == 'processDynNodes') return nodes(parent, callback);

}
````

A cleaner solution would IMO be better.

BTW I'm using aria-tabs and discovered this issue when I was about to do release. Thanks for aria-tabs etc.

**Comment by schrotie** [comment]
Thanks @nevf ! Do you need this like yesterday or is it not pressing for you right now? Weird, that I haven't seen this ... have used this in various minified stuff.

**Comment by nevf** [comment]
@schrotie No hurry, I've made the changes in a copy of the file so I'm ok. FYI I'm using parcel.js as the bundler.

**Comment by nevf** [comment]
@schrotie FYI the app I mentioned is live - thought you might want to see your aria-tabs at work. App https://clibu-notes-alpha-0-20-21.netlify.com and Blog post https://blog.clibu.com/2019/09/24/clibu-notes-alpha-demo-release-v0-20-21/

I've made some changes to get ```aria-tabs``` to better meet my needs. For example I've added options so it doesn't try and restore a tab when started and an option to notify the client a tab has been selected without activating it etc.  

I'm using [Lit-element](https://lit-element.polymer-project.org/) extensively and may either rewrite aria-tabs as a Lit-element web component or replace it with a Tab widget I largely wrote and am using in [Clibu](clibu.com). The later lets you re-arrange Tabs using drag and drop, but has no aria support.
